infrastructure-playbooks/rolling_update.yml

```diff
@@ -816,7 +845,7 @@
       when: containerized_deployment
 
     - name: non container | disallow pre-nautilus OSDs and enable all new nautilus-only functionality
-      command: ceph osd require-osd-release nautilus
+      command: "ceph --cluster {{ cluster|default(ceph) }} osd require-osd-release nautilus"
```

Comment:
```suggestion
      command: "ceph --cluster {{ cluster|default('ceph') }} osd require-osd-release nautilus"
```

Reply:
`cluster` is defined anyway in ceph-defaults, so I don't see how it could be undefined
if you wanna keep that `|default(ceph)` you must add quotes otherwise `ceph` will be interpreted as an undefined variable